seismostats/analysis/bvalue/more_positive.py

```diff
@@ -145,6 +146,52 @@ def _estimate(self) -> float:
                               delta_m=self.delta_m,
                               weights=self.weights)
 
+    def _std_bootstrap(self, n: int = 500, random_state: int = None) -> float:
+        '''
+        Bootstrap uncertainty of the b-value estimate. Depending on the number
+        of the magnitudes, this can take a while to compute.
+        '''
+        self._is_estimated()
```

Comment:
you are calling "_is_estimated" before running the function, no need to call it again.